terraform/opt-out-s3/main.tf

```diff
@@ -0,0 +1,53 @@
+# Temporary S3 bucket for opt-out
+
+data "aws_caller_identity" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+}
+
+resource "aws_s3_bucket" "opt_out" {
+  bucket = "${local.account_id}-ab2d-opt-out-temp"
```

Comment:
May I ask you, why did you use local.account_id in the name?

Reply:
It's common practice to use the account number in the bucket name to ensure a unique ID since bucket names share a global namespace.

Reply:
Using a region-specific codename prefix might be another idea for resource name collision avoidance, in the event for example a primary region-wide outage forced us to have to provision everything in a secondary region.